contracts/Nodes.sol

```diff
@@ -317,7 +317,9 @@ contract Nodes is AccessManagedUpgradeable, INodes {
         nodeId = _activeNodesAddressToId.get(nodeAddress);
     }
 
-    function getPassiveNodeIds(address nodeAddress) external view override returns (NodeId[] memory nodeIds) {
+    function getPassiveNodesIdsForAddress(
```

Comment:
For long function declarations, it is recommended to drop each argument onto its own line at the same indentation level as the function body. The closing parenthesis and opening bracket should be placed on their own line as well at the same indentation level as the function declaration.
https://docs.soliditylang.org/en/latest/style-guide.html